lib/rubocop/formatter/disabled_config_formatter.rb

```diff
@@ -126,6 +125,13 @@ def output_cop_comments(output_buffer, cfg, cop_name, offense_count)
         output_cop_param_comments(output_buffer, params, default_cfg)
       end
 
+      def support_autocorrect?(cop_name)
+        cop_class = Cop::Cop.registry.find_by_cop_name(cop_name)
+        return false unless (cop_instance = cop_class&.new)
+
+        cop_instance.respond_to?(:support_autocorrect?) && cop_instance.support_autocorrect?
```

Comment:
How come we have cops that don't support this method?

Reply:
`support_autocorrect?` is a class method, so `new` was not needed 💦

Reply:
This is my mistake, I didn't see it. This is new since the Cop refactor.
Thanks @koic